adm_program/system/common.php

```diff
@@ -38,6 +38,15 @@
     // => EXIT
 }
 
+// check for empty db and redirect to installation wizard
+try {
```

Comment:
Do we want to check for an empty db in every script? Why not check it only in index.php ?

Reply:
because in `adm_program/system/common.php` on Line 35 we have already the `config.php` loaded and opened the db connection.